### PR TITLE
[feat] Adding install wizard when config isn't set

### PR DESCRIPTION
**Pull request by zooley**

### CODE REVIEW

File: core/bootstrap/Install/Providers/WizardServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+ */
+
+namespace Bootstrap\Install\Providers;
+
+use Hubzero\Base\Middleware;
+use Hubzero\Http\Request;
+use Hubzero\Base\Traits\ErrorBag;
+
+/**
+ * Wiard service provider
```
Comment (zweidner) [inline core/bootstrap/Install/Providers/WizardServiceProvider.php]: typo

File: core/bootstrap/Install/Providers/WizardServiceProvider.php [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+		{
+			$this->app->redirect($this->app['request']->root());
+			//return $response;
+		}
+
+		$referrer = $request->getVar('REQUEST_URI', '', 'server');
+
+		if ($request->method() == 'POST' && \Hubzero\Utility\Uri::isInternal($referrer))
+		{
+			$data = $request->getVar('database', array(), 'post');
```
Comment (anthonyfuentes) [inline core/bootstrap/Install/Providers/WizardServiceProvider.php]: Should this line be changed to `getArray`?

File: core/bootstrap/Install/Wizard/index.php [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+ *
+ * @package   hubzero-cms
+ * @author    Shawn Rice <zooley@purdue.edu>
+ * @copyright Copyright 2005-2015 HUBzero Foundation, LLC.
+ * @license   http://opensource.org/licenses/MIT MIT
+ */
+
+defined('_HZEXEC_') or die();
+
+if ($msg = App::get('request')->getVar('msg'))
```
Comment (anthonyfuentes) [inline core/bootstrap/Install/Wizard/index.php]: Should this line be `getString`?

File: core/bootstrap/Install/config/app.php [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+	/*
+	|--------------------------------------------------------------------------
+	| Secret
+	|--------------------------------------------------------------------------
+	|
+	| This is an auto-generated, unique alphanumeric code for every
+	| installation. It is used for security functions.
+	|
+	*/
+	'secret' => 'lskafadfjlasdjfasdfaswer',
```
Comment (zweidner) [inline core/bootstrap/Install/config/app.php]: Should this be regenerated when writing the configs back to /app/config?

**Pull request merged**